src/components/fields/Info.js

```diff
@@ -3,7 +3,11 @@ import React, {Component} from 'react';
 
 export default class Info extends Component {
   render() {
-    return <Field {...this.props}>{this.props.children}</Field>;
+    return (
+      <Field {...this.props}>
+        <div className="js-test-info">{this.props.children}</div>
+      </Field>
```

Comment:
If I add the class on Field, it gives me 2 js-test-info classes, one on the localized wrapper one on Field